tests/utils/test_parallel_embeddings.py

```diff
@@ -474,3 +476,67 @@ def test_chimera_on_pegasus(self):
         m_nice = n_nice = m - 1  # A nice subgraph is relevant to Chimera sublattices:
         expected_number_of_cells = (m_nice // m_sub) * (n_nice // 3) * 3
         self.assertEqual(len(embeddings), expected_number_of_cells)
+
+    def test_T_family_T_kwargs(self):
+        # Like test basic, but pass family information
+        with_Ts = [True, False]
+        with_Tfamily = [True, False]
+        for topology, with_T, with_Tfamily in product(
+            ["chimera", "pegasus", "zephyr"], with_Ts, with_Tfamily
+        ):
+            T_family = topology
+            if topology == "chimera":
+                min_sublattice_size = 1
+                S = dnx.chimera_graph(min_sublattice_size)
+                T = dnx.chimera_graph(min_sublattice_size + 1)
+                num_emb = 4
+                tile = None
+                T_kwargs = {}
+            elif topology == "pegasus":
+                min_sublattice_size = 2
+                S = dnx.pegasus_graph(min_sublattice_size, nice_coordinates=True)
+                tile = S
+                T = dnx.pegasus_graph(min_sublattice_size + 1, nice_coordinates=True)
+                num_emb = 2
+                T_kwargs = {
+                    "nice_coordinates": True,
+                    "edge_list": list(T.edges),
+                    "node_list": list(T.nodes),
+                }
+            elif topology == "zephyr":
+                min_sublattice_size = 1
+                S = dnx.zephyr_graph(min_sublattice_size, coordinates=True)
+                tile = S
+                T = dnx.zephyr_graph(min_sublattice_size + 1, coordinates=True)
+                num_emb = 2
+                T_kwargs = {"coordinates": True}
+            if with_Tfamily is False:
+                T_family = None
+                T_kwargs = None  # Ignored in any case.
+            else:
+                # Cast T as standard nx.Graph; make sure propagation is correct:
+                T_kwargs["m"] = T.graph.get("rows")
+                T = nx.from_edgelist(T.edges)
+            if with_Ts is False:
```

Comment:
Wrong variable, that's why the bug for `T=None` is never caught with this test.
```suggestion
            if with_T is False:
```
Or, change `with_T` to `with_Ts` in the for loop above.

Overall, though, I'd use `parameterize` over the test case, not a loop.

Reply:
Added the code suggestion for parameterized use.